frontend/src/login/LoginConsole.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { Button, Container, makeStyles, TextField } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  form: {
+    width: "100%",
+    marginTop: theme.spacing(1)
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2)
+  }
+}));
```

Comment:
Why did you choose to do `useStyles` instead of defining and using classes in `index.css`?

Reply:
See my other comment